src/auditlog/models.py

```diff
@@ -232,6 +237,73 @@ def changes_str(self, colon=': ', arrow=smart_text(' \u2192 '), separator='; '):
 
         return separator.join(substrings)
 
+    @property
+    def changes_display_dict(self):
+        """
+        :return: The changes recorded in this log entry intended for display to users as a dictionary object.
+        """
+        # Get the model
+        model = self.content_type.model_class()
+        changes_display_dict = {}
+        # grab the changes_dict and iterate through
+        for field_name, values in iteritems(self.changes_dict):
+            # try to get the field attribute on the model
+            try:
+                field = model._meta.get_field(field_name)
+            except FieldDoesNotExist:
+                # use the reverse field mapping to get the field
+                from auditlog.registry import auditlog
+                model_fields = auditlog.get_model_fields(model._meta.model)
+                reversed_field = model_fields['reverse_mapping_fields'].get(field_name)
+                field = model._meta.get_field(reversed_field)
```

Comment:
Again, a FieldDoesNotExist can be raised, for example when a model is changed.

Reply:
Ok, with the update to `mapping_fields` not changing the `diff.py` output, I didn't need the `reverse_mapping_fields`. I have now removed that section, and in the case of `FieldDoesNotExist` fall back on the `field_name` auditlog has stored.